prow/scripts/governance.sh

```diff
@@ -7,7 +7,7 @@ readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPT_DIR}/library.sh"
 
 readonly ARGS=("$@")
-readonly MILV_IMAGE="magicmatatjahu/milv:0.0.6"
+readonly MILV_IMAGE="magicmatatjahu/milv:0.0.7-alpha"
```

Comment:
How stable is `alpha`? Did you test it against all of our repositories where governance is enabled?

Reply:
`alpha` has the ability to ignore checking internal links in given files/folders and I tested it in some example cases, but before release `0.0.7` version I want to add a few features that do not affect the task.

And yes, I tested new version in all of our repositories where governance is enabled 😸

Reply:
Perfect 💯 Approved 👏

Reply:
BTW @magicmatatjahu why did you name the version `0.0.7-alpha`, not `0.0.7`, if everything is well-tested? 😄

Reply:
@pkosiec `...but before release 0.0.7 version I want to add a few features that do not affect the task.` 😄